src/quill-cursors/touch-device.ts

```diff
@@ -0,0 +1,4 @@
+export function isTouchDevice(): boolean {
+  return (('ontouchstart' in window) ||
```

Comment:
Is this the normal way to determine this? How does this behave on eg a touchscreen laptop with a mouse attached?

I'd have thought using the [`pointer`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/pointer) or [`hover`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/hover) media selectors (or both?!) would be a better way to determine this...?